sdk/search/azure-search-documents/src/main/java/com/azure/search/documents/indexes/models/EdgeNGramTokenFilter.java

```diff
@@ -3,19 +3,15 @@
 
 package com.azure.search.documents.indexes.models;
 
-import com.azure.core.annotation.Fluent;
 import com.fasterxml.jackson.annotation.JsonProperty;
-import com.fasterxml.jackson.annotation.JsonTypeInfo;
-import com.fasterxml.jackson.annotation.JsonTypeName;
 
 /**
  * Generates n-grams of the given size(s) starting from the front or the back
  * of an input token. This token filter is implemented using Apache Lucene.
  */
-@JsonTypeInfo(use = JsonTypeInfo.Id.NAME, include = JsonTypeInfo.As.PROPERTY, property = "@odata.type")
-@JsonTypeName("#Microsoft.Azure.Search.EdgeNGramTokenFilter")
-@Fluent
 public final class EdgeNGramTokenFilter extends TokenFilter {
+    @JsonProperty(value = "@odata.type")
```

Comment:
Do we need the JSON property? Is this class every directly used in serialization?

Reply:
You are right, we set the private field instead of json serialization. Will remove it here